syn/tcl/yosys_run_synth.tcl

```diff
@@ -30,6 +30,17 @@ yosys "chparam -set RV32M $lr_synth_ibex_multiplier $lr_synth_top_module"
 
 yosys "chparam -set RegFile $lr_synth_ibex_regfile $lr_synth_top_module"
 
+if { $lr_synth_ibex_secure_ibex } {
+  yosys "chparam -set SecureIbex 1 $lr_synth_top_module"
+  # Place keep_hierarchy contraints on relevant modules to prevent aggressive
+  # synthesis optimzations across the boundaries of these modules.
+  yosys "hierarchy -check -top $lr_synth_top_module"
+  yosys "setattr -mod -set keep_hierarchy 1 *prim_generic_and2*"
+  yosys "setattr -mod -set keep_hierarchy 1 *prim_generic_buf*"
+  yosys "setattr -mod -set keep_hierarchy 1 *prim_generic_clock_mux2*"
+  yosys "setattr -mod -set keep_hierarchy 1 *prim_generic_flop*"
+}
+
```

Comment:
I think you should again remove these constraints before the very final flattening step, no? Can you compare with the AES setup please?